docs/pages/kubernetes-access/discovery/google-cloud.mdx

```diff
@@ -0,0 +1,578 @@
+  or another platform
+
+<Tabs dropdownCaption="How your host is running:" 
+dropdownSelected="Google Cloud">
+<TabItem options="Google Cloud" label="Package Manager ">
+
+On the host where you will run the Teleport Kubernetes Service and Discovery
+Service, start the Teleport service:
+
+```code
+$ sudo systemctl start teleport
+```
+</TabItem>
+<TabItem options="Google Cloud" label="TAR Archive ">
+
+On the host where you will run the Teleport Kubernetes Service and Discovery
+Service, create a systemd service configuration for Teleport, enable the
+Teleport service, and start Teleport:
+
+```code
+$ sudo teleport install systemd -o /etc/systemd/system/teleport.service
+$ sudo systemctl enable teleport
+$ sudo systemctl start teleport
+```
+</TabItem>
+<TabItem options="Other Platform" label="Package Manager">
+
+When you installed Teleport via package manager, the installation process
+created a configuration for the init system `systemd` to run Teleport as a
+daemon. 
+
+This service reads environment variables from a file at the path
+`/etc/default/teleport`. Teleport's built-in Google Cloud client reads the
+credentials file at the location given by the `GOOGLE_APPLICATION_CREDENTIALS`
+variable. 
+
+Ensure that `/etc/default/teleport` has the following content:
+
+```
+GOOGLE_APPLICATION_CREDENTIALS="/var/lib/teleport/google-cloud-credentials.json"
+```
+
+Start the Teleport service:
+
+```code
+$ sudo systemctl start teleport
+```
+</TabItem>
+<TabItem options="Other Platform" label="TAR Archive">
+
+On the host where you are running the Teleport Discovery Service and Kubernetes
+Service, create a systemd configuration that you can use to run Teleport in the
+background:
+
+```code
+$ sudo teleport install systemd -o /etc/systemd/system/teleport.service
+$ sudo systemctl enable teleport
+```
+
+This service reads environment variables from a file at the path
+`/etc/default/teleport`. Teleport's built-in Google Cloud client reads the
+credentials file at the location given by the `GOOGLE_APPLICATION_CREDENTIALS`
+variable. 
+
+Ensure that `/etc/default/teleport` has the following content:
+
+```
+GOOGLE_APPLICATION_CREDENTIALS="/var/lib/teleport/google-cloud-credentials.json"
+```
+
+Start the Discovery Service and Kubernetes Service:
+
+```code
+$ sudo systemctl start teleport
+```
+
+</TabItem>
+</Tabs>
+
+## Step 3/3. Connect to your GKE cluster
+
+### Allow access to your Kubernetes cluster
+
+Ensure that you are in the correct Kubernetes context for the cluster you would
+like to enable access to:
+
+```code
+$ kubectl config current-context
+```
+
+<Details title="Using the wrong context?">
+
+Retrieve all available contexts:
+
+```code
+$ kubectl config get-contexts
+```
+
+Switch to your context, replacing `CONTEXT_NAME` with the name of your chosen
+context:
+
+```code
+$ kubectl config use-context CONTEXT_NAME
+Switched to context CONTEXT_NAME
+```
+
+</Details>
+
+(!docs/pages/includes/kubernetes-access/rbac.mdx!)
```

Comment:
It's not part of this PR but I noticed it while reviewing this doc... This Partial uses a Var component to set a shell Var to then use in a single command:

````
```code
# e.g., myuser
$ TELEPORT_USER=<Var name="teleport-user"/>
$ tctl get user/${TELEPORT_USER?} > user.yaml
```
````

Why not just have the command use the Var component value?

````
```code
# e.g., myuser
$ tctl get user/<Var name="teleport-user"/> > user.yaml
```
````